extensions/default/src/Panels/PanelStudyBrowser.jsx

```diff
@@ -247,7 +247,7 @@ function _mapDisplaySets(displaySets, thumbnailImageSrcMap) {
 
     array.push({
       displaySetInstanceUID: ds.displaySetInstanceUID,
-      description: ds.SeriesDescription,
+      description: ds.SeriesDescription || 'Unspecified',
```

Comment:
Series description is sometimes null, and that was causing an exception rendering the thumbnails.

Reply:
Can you do so for `extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx` as well?

Reply:
Reading the next comment (inside `getSopClassHandlerModule` I don't understand why we need `Unspecified` again here. Wouldn't `makeDisplaySet` handles this?

Reply:
Maybe it should, but I was seeing exceptions getting thrown.  Maybe it should be in getSopClassHandlerModule to put it in a single place, but I would need a bit of help figuring out exactly where.